onnxruntime/core/mlas/lib/compute.cpp

```diff
@@ -880,11 +882,12 @@ Return Value:
 #else
         float Maximum = MlasReduceMaximumF32Kernel(Input, D);
 #endif
-        float NegativeMaximum = -Maximum;
-        if (SmoothSoftmax && NegativeMaximum > 0.0f) {
-            NegativeMaximum = 0.0f;
+        if (SmoothSoftmax && Sink > Maximum) {
+            Maximum = Sink;
         }
```

Comment:
What the value of parameter `SmoothSoftmax` should be, when per head smooth softmax is used?

Reply:
It is true when either the attirbute is set or head_sink is provided. The sink value head_sink of current head, or 0 if head_sink is not provided:

https://github.com/microsoft/onnxruntime/blob/60de045cdea5dc3822f5e680d9559c712e183d5f/onnxruntime/contrib_ops/cpu/bert/gqa_attention_base.h#L315-L317